lib/core/src/switcher/switcher.tsx

```diff
@@ -19,7 +19,8 @@ export interface SwitcherOption<T> {
 	 */
 	iconLabel?: ButtonProps["iconLabel"];
 	/**
-	 * The React's key for the option
+	 * The React's key for the option. It's recommend to explicitly define
```

Comment:
It's recommended

Reply:
Thank you! I'll run gramma rly again I think